.circleci/wait-for-job-finish.sh

```diff
@@ -18,8 +18,9 @@ while [ "$counter" -le 1500 ]; do
     exit 0
   fi
 
-  sleep 10
-  counter=$(($counter + 10))
+  echo "Checking job status: Waiting for parallel running nodes to finish. Waited $counter seconds."
```

Comment:
Suggest removing this (and restoring the `no_output_timeout` configuration) since this inflates the logs without really providing useful information.

```suggestion
```